docs/Operators.md

```diff
@@ -139,6 +139,9 @@
   * <sub>experimental</sub> <a href="#ScaledTanh">ScaledTanh</a>
   * <sub>experimental</sub> <a href="#ThresholdedRelu">ThresholdedRelu</a>
 
+  **Operators with function registered:**
```

Comment:
Change this to ""Functions""

Reply:
Only call it Functions may be misleading (trust me, users may get confused by functions again), I think Operators with function registered is Okay. Because they are operators, right?